java-checks/src/main/java/org/sonar/java/checks/aws/AwsLongTermAccessKeysCheck.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * SonarQube Java
+ * Copyright (C) 2012-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.java.checks.aws;
+
+import org.sonar.check.Rule;
+import org.sonar.java.checks.methods.AbstractMethodDetection;
+import org.sonar.plugins.java.api.semantic.MethodMatchers;
+import org.sonar.plugins.java.api.tree.NewClassTree;
+
+@Rule(key = "S6263")
+public class AwsLongTermAccessKeysCheck extends AbstractMethodDetection {
+
+  private static final String MESSAGE = "Make sure using a Long-term access key is safe here.";
```

Comment:
I am seeing this in the rspec and the ticket as well but should "Long" be capitalized here? Does it refer to some kind of IP on a technology?